src/foraging_gui/Foraging.py

```diff
@@ -338,7 +354,7 @@ def connectSignalsSlots(self):
         )
         self.ID.returnPressed.connect(
             lambda: self.AutoTrain_dialog.update_auto_train_fields(
-                subject_id=self.ID.text(),
```

Comment:
@micahwoodard So you are replacing all calls to `ID.text()` with the schema. I would make sure to check other files. It looks like Dialogs.py calls ID.text() in at least one place.

Reply:
Ah yes really good point for probably all of the replaced variables

Reply:
Checked widgets being referenced outside of Foraging.py and changed ones that were referencing the information and left the ones interacting with actual widget